utils/gear-logger/src/index.ts

```diff
@@ -0,0 +1,2 @@
+import { logger } from './logger';
+export { logger };
```

Comment:
```suggestion
export { logger } from './logger';
```
Actually we can only keep `index.ts`